meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -176,6 +176,67 @@ python() {
         bb.fatal("MENDER_PARTITION_ALIGNMENT_MB is deprecated. Please define MENDER_PARTITION_ALIGNMENT_KB instead.")
 }
 
+addhandler mender_vars_handler
+python mender_vars_handler() {
+    if(isinstance(e, bb.event.ParseCompleted)):
+        from bb import data
+        import os
+        import re 
+        import json
+
+        path = d.getVar("LAYERDIR_MENDER")
+        path = os.path.join(path, "conf/mender-vars.json")
+
+        if( (os.path.isfile(path))):
```

Comment:
In general Python doesn't need parentheses around if statements. This goes for the whole snippet.

Reply:
haha, didn't even notice :see_no_evil: